### PR TITLE
Feature/home page content

### PR DESCRIPTION
**Pull request by GeBon22**

### CODE REVIEW

**Review by actyralf — CHANGES_REQUESTED** [review]
Codewise it looks good, however I can't see the animations in preview and it throws errors in Dev tools. Please have a look into that...

**Comment by GeBon22** [comment]
i tried to make the lottie animation files visible on Vercel --> I decided to move forward and start working on the other user stories and return to this odd 'Error' at a later time. Please consider this when reviewing. Thanks

File: pages/index.js [hunk 1]
```diff
@@ -1,26 +1,121 @@
...
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+/* const BestellungContainer = styled.div`
+display: flex;
+flex-direction: column;
+justify-content: space-evenly;
+align-items: flex-start;
+` */
```
Comment (stefankorder) [inline pages/index.js]: Please delete this unused code.

**Review by stefankorder — APPROVED** [review]
Good work! Please take a look at my comment.

File: pages/index.js (hunk 1 above)
Comment (GeBon22) [reply pages/index.js]: check

**Pull request merged**